tutorial-detailed-samples.md

```diff
@@ -401,23 +376,23 @@ _Edit Sample_ page for a single sample, click the sample's name or alias on the
 To bulk edit a set of samples, click the checkboxes next to them on the _Samples_ page and click
 _Edit_ at the top left of the table.
 
-For attributes which cannot be changed via MISO (e.g. Sample Class), please email gsi@oicr.on.ca
-or file a JIRA ticket in GSI Common to get assistance from the MISO team.
+For attributes which cannot be changed via MISO (e.g. Sample Class), {{ site.miso_admin_contact }} for help. This may
```

Comment:
Does site.miso_admin_contact include a verb or would this evaluate to 'For attributes which cannot be changed via MISO, [email address] for help'?

Reply:
Includes a verb. See `_config.yml`: "email gsi@oicr.on.ca or file a JIRA ticket in _GSI Common_"